### DISCUSSION TITLE
GPX saving issue in Zepp on Huawei phones

### DISCUSSION THREAD

**Discussion by evilweiss1** [zepp-health/zeppos-forum#375] [Q&A]
I have a problem with saving GPX or KML tracks in Zepp specifically with Huawei P60 Pro. And I know at least one the same issue with another P60 Pro and P70 pro. I use file manager to open GPX file, track appears in Zepp, but Save button tap do nothing. It work fine on other phones, like pixel 6a. Looks like Save button not working on Huawei phones specifically. I tried different Zepp versions, e.g. 8.9.0, 8.11.5, 8.13.1 or latest 9.0.0. Problem exists on all versions of Zepp tried. Saving of gpx is working on the old 7.6.1 version from huawei gallery, but the map is not working there and Zepp can not send saved track to t-rex 2, I cant use it. I cant find any solution or even the same problem in internet.

https://github.com/user-attachments/assets/8b327736-7e5b-4400-9b60-a674555f06f0

Amazfit support answer: I got the reply from the team, the R&D team uses the files provided by users and can save successfully without any recurrence. Can you please uninstall and reinstall the latest online app version to try again?

**Comment by MyoungXUE** [comment]
Hello, I just noticed this problem. Does the latest version of Zepp still have this problem?

You can send me the GPX together, and I will try to reproduce your question.

**Comment by evilweiss1** [comment]
Hello, I tried the latest 9.0.5 version of zepp, "save" button is still not
working for any *.gpx files. See attached for example. I know that saving
is NOT working for several p60 pro and pura 70. And working fine for Pixel
6a or Honor 9x. Amazfit support wrote to me: "I got the reply from the
team, the R&D team uses the files provided by users and can save
successfully without any recurrence".

**Comment by evilweiss1** [comment]
[New track.zip](https://github.com/user-attachments/files/17552085/New.track.zip)

**Comment by MyoungXUE** [comment]
Hello, a colleague noticed that you have previously reported this issue and is still trying to reproduce it. Please be patient and wait for the technical personnel to fix it.
